### ISSUE TITLE
[16.0] mail_debrand > Only removing href on languages other than english

### ISSUE THREAD

**Issue by FerranHotaka** [OCA/social#1237]
The addon leaves text present on emails built with templates that are sent to customers whose language isn't set to english. In spanish, the text string that is left is "Con tecnología de" as it's present in the .po file as follows:

```
#. modules: common_connector_library, mail, digest, portal
#: model_terms:ir.ui.view,arch_db:common_connector_library.connector_digest_mail_main
#: model_terms:ir.ui.view,arch_db:digest.digest_mail_main
#: model_terms:ir.ui.view,arch_db:mail.mail_notification_layout
#: model_terms:ir.ui.view,arch_db:mail.mail_notification_light
#: model_terms:ir.ui.view,arch_db:portal.portal_record_sidebar
msgid "Powered by"
msgstr "Con tecnología"

#. module: web
#: model_terms:ir.ui.view,arch_db:web.brand_promotion_message
msgid "Powered by %s%s"
msgstr "Con tecnología de %s%s"

#. module: web
#: model_terms:ir.ui.view,arch_db:web.login_layout
msgid "Powered by <span>Odoo</span>"
msgstr "Respaldado por <span>Odoo</span>"
```

This doesn't seem to happen on mesages sent with the "Send message" button on the side menu, I have replicated it with the "Sales: Send Quotation" template and the "Sales: Order Cancellation" one.

Any suggestions?

**Comment by StefanRijnhart** [comment]
Please have a look at https://github.com/OCA/social/pull/1272

**Comment by StefanRijnhart** [comment]
Fixed in 16.0 with #1272
